spec/css/selector.hrx

```diff
@@ -55,6 +55,58 @@
   c: d;
 }
 
+<===>
+================================================================================
+<===> attribute/quoted_non_identifier/input.scss
+// Quotes should be preserved when the string they contain is not an identifier.
+// See https://github.com/sass/dart-sass/issues/598.
+[a="b."] {c: d}
+
+<===> attribute/quoted_non_identifier/output.css
+[a="b."] {
+  c: d;
+}
+
+<===>
+================================================================================
+<===> placeholder/pseudoselectors/options.yml
+---
+:todo:
+- sass/libsass#2422
+
+<===> placeholder/pseudoselectors/matches/input.scss
+// Style rules with placeholders should only be deleted if they can't match
+// *any* real elements.
+a:matches(%b) {w: x}
+a:matches(%b, c) {y: z}
```

Comment:
These should be [separate test cases](https://github.com/sass/sass-spec/blob/master/STYLE_GUIDE.md#do-test-only-one-thing-per-spec). Same for the `:not` tests.